core/templates/domain/editor/undo_redo/change.model.ts

```diff
@@ -36,7 +36,7 @@ import { WorkedExampleBackendDict } from
 import { Collection } from 'domain/collection/collection.model';
 import { Question } from 'domain/question/QuestionObjectFactory';
 import { Skill } from 'domain/skill/SkillObjectFactory';
-import { Story } from 'domain/story/StoryObjectFactory';
+import { Story } from 'domain/story/story-object.model';
```

Comment:
Here and elsewhere, I think it might be better to refer to the class as a "story model" rather than a "story object model".

Reply:
Has this been addressed? I still see "story object model" here and below. Please make sure to reply to comment saying "done" or similar so that it is easier for the reviewer to know if they have been addressed or not.

Reply:
> Has this been addressed? I still see "story object model" here and below. Please make sure to reply to comment saying "done" or similar so that it is easier for the reviewer to know if they have been addressed or not.

I referred to this [doc ](https://github.com/oppia/oppia/wiki/Angular-Migration#5-rename-the-file)to rename the files. Which states For example, PlaythroughObjectFactory.ts should be renamed to playthrough-object.model.ts

Reply:
Ah I see, hm, I think we should standardise this but it can be done separately, let's drop the "object" for story models in this PR.

Reply:
@kevintab95 Done!